docs/5.0/preview/upcoming-releases.md

```diff
@@ -43,10 +49,30 @@ You can find the full list of fixes and features in the
 | Database Access                     | All               | SSO into PostgreSQL or MySQL. Use AWS Aurora RDS or on-premises. Read [more here](./database-access.md).
 | OSS RBAC                            | OSS               | Open source role based access controls. Check out the [design doc](https://github.com/gravitational/teleport/blob/master/rfd/0007-rbac-oss.md) and [issue 4136](https://github.com/gravitational/teleport/issues/4136).
 | Terraform Provider                  | All               | Configure Teleport without UI using Terraform provider. More details [here](https://github.com/gravitational/teleport-plugins/projects/3#card-49866475).
+| Client libraries and API            | All               | Use Go to create access workflows. Review the [design doc](https://github.com/gravitational/teleport/pull/4746) and [issue 4763](https://github.com/gravitational/teleport/issues/4763).
```

Comment:
For the Client libraries and API work, the following is what we're on track for.

6.0

1. Auth GRPC API moved to `api` package with limited dependencies.
2. Additional CRUD endpoints needed for Terraform provider.
3. Support connecting to Auth through Proxy (needed for Cloud).
4. Credential loader.

6.1

1. Move Access requests code into client from `teleport-plugins`.
2. Provide simplified high level access hooks.

Reply:
@russjones can you remove things that we are not going to deliver from 4673 and create a new issue in 6.1 with extra deliverables

Reply:
6.0

https://github.com/gravitational/teleport/issues/5464

6.1

https://github.com/gravitational/teleport/issues/5465
https://github.com/gravitational/teleport/issues/5466